OpenMEEG/src/assemble.cpp

```diff
@@ -81,17 +86,11 @@ int main(int argc, char** argv)
     /*********************************************************************************************
     * Computation of Head Matrix for BEM Symmetric formulation
     **********************************************************************************************/
-    if ( ( !strcmp(argv[1], "-HeadMat") ) | ( !strcmp(argv[1], "-HM" ) ) | ( !strcmp(argv[1], "-hm") ) ) {
-        if ( argc < 3 ) {
-            std::cerr << "Please set geometry filepath !" << endl;
-            exit(1);
-        }
-        if ( argc < 4 ) {
-            std::cerr << "Please set conductivities filepath !" << endl;
-            exit(1);
-        }
+    if ( option(argv[1], {"-HeadMat","-HM", "-hm"}) ) {
         if ( argc < 5 ) {
-            std::cerr << "Please set output filepath !" << endl;
+            cerr << "Please set geometry filepath" << endl;
+            cerr << "           conductivities filepath" << endl;
+            cerr << "           output filepath" << endl;
             exit(1);
```

Comment:
agreed. It's good practice to say what was wrong instead of just saying how it should be.